section3/set11/exercises/11-average/main.go

```diff
@@ -60,22 +60,18 @@ func main() {
 			fmt.Println("Please tell me numbers (maximum 5)")
 			return
 		}
-
 		avg[i] = n
 	}
-	fmt.Printf("Array: %v\n", avg)
+	//fmt.Printf("Array: %v\n", avg)
 
 	sum := 0
 	// create for loop to prompt user to input values
 	for i := 0; i < len(os.Args)-1; i++ {
-		if i == 4 {
-			break
-		}
-
-		presum := avg[i+1]
-		sum = presum + avg[i+2]
+		sum += avg[i]
 	}
+	//fmt.Printf("Sum: %d\n", sum)
 
 	// avg the array using a for loop
-	fmt.Printf("Sum: %d\n", sum)
+	fmt.Printf("Your array: %v\n", avg)
+	fmt.Printf("Average: %d\n", sum/len(os.Args)-1)
```

Comment:
use avg array for length

Reply:
average has type of [5]int because it is an array, while sum has a length of int, since it is a regular integer. So they wouldn't be comparable to each other. dividing by 5 is a better idea because the program needs to have 5 inputs, because that is listed in the instructions.

Reply:
error has been fixed